backend/apps/organizations/serializers.py

```
@@ -1,7 +1,31 @@
 from rest_framework import serializers
-from .models import Organization
+
+from .models import Organization, OrganizationMembership
+
+
+class OrganizationMembershipSerializer(serializers.ModelSerializer):
+    username = serializers.CharField(source="user.username", read_only=True)
+
+    class Meta:
+        model = OrganizationMembership
+        fields = ["id", "user", "username", "role", "joined_at"]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)